### PR TITLE
Added missing default sort by primary key(s)

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Please fill your PR of missing parts indicated by the "Needs" labels.

Feel free to ask if you want some precision of what is needed.

**Comment by greg0ire** [comment]
@Soullivaneuh done

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,3 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 ### Added
+
+- The primary key(s) is / are now appended to the to the list of fields used
```
Comment (Soullivaneuh) [inline CHANGELOG.md]: Could you shorten this? The change-log has to be clean and concise, focused on the action.

Something like: `Added default sort for primary key(s)`.

Comment (Soullivaneuh) [inline CHANGELOG.md]: I'm wondering if this should not be on the `Fixed` section...

Comment (greg0ire) [inline CHANGELOG.md]: @Soullivaneuh done

**Comment by greg0ire** [comment]
@Soullivaneuh Added that to the Fixed section, because it was marked a patch. No bug was reported, but could have been. Gray area I guess…

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,3 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 ### Added
+
+### Fixed
+
+- Added default sort by primary key(s).
```
Comment (Soullivaneuh) [inline CHANGELOG.md]: `Added missing default`

Sounds more like a fix like that. :wink:

Comment (greg0ire) [inline CHANGELOG.md]: Indeed, fixed.

**Comment by Soullivaneuh** [comment]
Thank you @greg0ire.

**Pull request merged**